src/app/css/_applicationFrame.scss

```diff
@@ -7,53 +7,45 @@
 @import 'themes';
 
 .app {
+
     font-family: "Segoe UI" !important;
     font-size: 14px !important;
     display: grid;
-    width: 100%;
-    height: 100vh;
+    grid-template-rows: $mastheadHeight $navigationHeight 1fr;
     grid-template-columns: 1fr;
-    grid-template-rows: $mastheadHeight 1fr;
     display: -ms-grid;
+    -ms-grid-template-rows: $mastheadHeight $navigationHeight 1fr;
     -ms-grid-columns: 1fr;
-    -ms-grid-rows: 40px 1fr;
+    width: 100%;
+    height: 100vh;
+    overflow-y: hidden;
     @include themify($themes) {
         color: themed('textColor');
         background: themed('backgroundColor');
     }
 
-    .header {
+    .masthead {
+        width: 100%;
         grid-row: 1;
         grid-column: 1;
         -ms-grid-row: 1;
         -ms-grid-column: 1;
     }
-    .content {
+
+    .navigation {
+        min-height: 90px;
```

Comment:
Not sure if I missed, wonder if you would like to leave it min-height or make it fixed height

Reply:
yes -- looks like an oversight -- will fix.